File: client/src/core/ErrorDTO.ts
/**
 * Clase que representa un error en la aplicacion (ya sea local o remoto)
 *
 * @class ErrorDTO
 */
export class ErrorDTO {
  /**
   * Crea una instancia de ErrorDTO.
   * @param {string} code codigo de error (string parseable por el sistema)
   * @param {string} message descripcion del error (mensaje a mostrar al usuario)
   * @memberof ErrorDTO
   */
  constructor(public code: string, public message: string) {}
}

File: server/src/main/java/com/mmoscovich/webcontab/dto/ErrorDTO.java
package com.mmoscovich.webcontab.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * DTO que representa un error
 *
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class ErrorDTO {
	/** Codigo de error */
	private String code;
	
	/** Descripcion del error */
	private String description;
}
